etc/helm/pachyderm/templates/pachd/idp-secret.yaml

```diff
@@ -0,0 +1,14 @@
+{{- /*
+SPDX-FileCopyrightText: Pachyderm, Inc. <info@pachyderm.com>
+SPDX-License-Identifier: Apache-2.0
+*/ -}}
+{{- if and ( not .Values.oidc.upstreamIDPsSecretName) (or .Values.oidc.upstreamIDPs .Values.oidc.mockIDP) }} 
+apiVersion: v1
+kind: Secret
+metadata:
+  name: pachyderm-identity
+  namespace: {{ .Release.Namespace }}
+stringData:
+  idps: |
+    {{ include "pachyderm.idps" . }}
```

Comment:
It's a little action-at-a-distance-y for `pachyderm.idps` to have to be indented by 4 spaces.  It does do that, but it makes more sense to me to do that here.